money-monkey-web/app/lib/validation.ts

```
@@ -1,14 +1,16 @@
+
 export const NewExpenseSchema: Yup.ObjectSchema<NewExpense> = Yup.object({
   name: Yup.string()
     .required("Required!")
     .min(1, "Too short!")
     .max(50, "Too long!"),
-  amount_cents: Yup.number()
+  amount: Yup.string()
     .required("Required!")
-    .min(1, "Enter at least $0.01!"),
+    .matches(/^\d+(\.\d{1,2})?$/, "Must be a valid amount with up to 2 decimal places")
+    .test("is-greater-than-minimum", "Enter at least $0.01!", value => parseFloat(value || "0") > 0.01),
```

Biome

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)